Source/Dafny/DafnyAst.cs

```diff
@@ -7932,6 +7932,18 @@ public IfStmt(IToken tok, IToken endTok, bool isBindingGuard, Expression guard,
       this.Thn = thn;
       this.Els = els;
     }
+    public IfStmt(IToken tok, IToken endTok, bool isBindingGuard, Expression guard, BlockStmt thn, Statement els, Attributes attrs)
```

Comment:
Since this is a new constructor, is the previous one like this but without the `attrs` argument? If so, are there any callers of the previous one?